.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: bash
+sudo: true
```

Comment:
I played a bit around our packages are not on the [whitelist](https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise). The second problem is we need to set the language to python to use pip. So the only solution which can work is writing a python script instead of the Makefile. But makes this sense?